4.1-beta-post

```diff
@@ -0,0 +1,43 @@
+### Mastodon (496/500 characters):
+
+JupyterLab 4.1 beta - call for testing! Included:
+- CSS customization,
+- diagrams in Markdown,
+- inline completer (for AI models),
+- better keyboard navigation,
+- execution history access,
+- links in tracebacks,
+- error indicators in ToC
+- plugin manager
+
+```
+pip install -U 'jupyterlab>=4.1.0b2'
+# or
+# conda install -c conda-forge/label/jupyterlab_beta -c conda-forge jupyterlab
```

Comment:
Maybe we can also add a Binder link to make it easier to test?

Reply:
Here is one: https://mybinder.org/v2/gist/krassowski/242109ec95e458052e48e41f5c7e969c/HEAD?urlpath=lab though you may need to reword to keep under character count

Reply:
Thanks! For X / Twitter it's part of a second post so it fits :+1: